src/main/java/org/zalando/nakadi/domain/Permission.java

```diff
@@ -43,11 +43,26 @@ public boolean equals(final Object o) {
         final Permission that = (Permission) o;
         return Objects.equals(resource, that.resource) &&
                 Objects.equals(operation, that.operation) &&
-                Objects.equals(authorizationAttribute, that.authorizationAttribute);
+                Objects.equals(authorizationAttribute.getDataType(), that.authorizationAttribute.getDataType()) &&
+                Objects.equals(authorizationAttribute.getValue(), that.authorizationAttribute.getValue());
     }
 
     @Override
     public int hashCode() {
         return Objects.hash(resource, operation, authorizationAttribute);
     }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder();
+        sb.append("Permission {");
+        sb.append("resource: ");
+        sb.append(resource);
+        sb.append("\tOperation: ");
+        sb.append(operation.toString());
+        sb.append("\tAuthorization attribute: ");
+        sb.append(authorizationAttribute.toString());
+        sb.append("}.");
```

Comment:
why not use chain calls here? `.append(...).append(...).append(...)`

Reply:
I guess I must have been distracted. Will fix this right away

Reply:
Fixed in 0c0d075